Pod/Classes/Swift/VOKCoreDataManager+Swift.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  VOKCoreDataManager+Swift.swift
+//  Vokoder
+//
+//  Created by Carl Hill-Popper on 1/17/16.
+//  Copyright © 2016 Vokal.
+//
+
+/**
+Swiftier versions of some Vokoder functions that return more exact managed object subclass types.
+*/
+public extension VOKCoreDataManager {
+    
+    /**
+     Create an appropriately typed instance of a given NSManagedObject subclass.
+     
+     - parameter objectClass: The class of object to create
+     - parameter context: The managed object context in which to create the object or nil for the main context (defaults to nil)
+     - returns: A new instance of the given class
+     */
+    public func managedObjectOfClass<T: NSManagedObject>(objectClass: T.Type,
+        inContext context: NSManagedObjectContext? = nil) -> T {
+            guard let result = self.managedObjectOfClass(objectClass, inContext: context) as? T else {
+                fatalError("Could not cast NSManagedObject to \(String(T))")
+            }
+            return result
+    }
+    
+    /**
+     Deserializes an array of dictionaries full of strings and creates (or updates) instances of a managed object subclass in the given context.
+     
+     - parameter inputArray: An array of dicionaries defining managed object subclasses
+     - parameter objectClass: The class of objects to create
+     - parameter context: The managed object context in which to create the objects or nil for the main context (defaults to nil)
+     - returns: A typed Array of created or updated objects
+     */
+    public func importArray<T: NSManagedObject>(inputArray: [[String : AnyObject]],
+        forClass objectClass: T.Type,
+        withContext context: NSManagedObjectContext? = nil) -> [T] {
+            guard let result = self.importArray(inputArray,
+                forClass: objectClass,
+                withContext: context) as? [T] else {
+                    fatalError("Could not cast array of NSManagedObjects into \(String(T))")
```

Comment:
Is there any way to add tests around this to see what happens if the import fails? If it doesn't return an empty array, It might be useful to have this throw an error rather than just dying in that case.